package.json

```diff
@@ -10,14 +10,15 @@
     "url": "git@github.com:sbugert/react-native-admob.git"
   },
   "dependencies": {
-    "prop-types": "^15.5.10"
+    "prop-types": "15.7.2"
   },
   "peerDependencies": {
-    "react-native": ">=0.44.0"
+    "react-native": "0.64.1"
```

Comment:
unless you need to use something specific to this version of react-native (which we don't use yet), I'd lower this to something like >=62 or something.